src/ert_storage/endpoints/experiments.py

```diff
@@ -104,6 +124,35 @@ def delete_experiment(*, db: Session = Depends(get_db), experiment_id: UUID) ->
     db.commit()
 
 
+PRIOR_FUNCTION_TO_PYDANTIC: Mapping[ds.PriorFunction, Type[js.Prior]] = {
+    ds.PriorFunction.const: PriorConst,
+    ds.PriorFunction.trig: PriorTrig,
+    ds.PriorFunction.normal: PriorNormal,
+    ds.PriorFunction.lognormal: PriorLogNormal,
+    ds.PriorFunction.ert_truncnormal: PriorErtTruncNormal,
+    ds.PriorFunction.stdnormal: PriorStdNormal,
+    ds.PriorFunction.uniform: PriorUniform,
+    ds.PriorFunction.ert_duniform: PriorErtDUniform,
+    ds.PriorFunction.loguniform: PriorLogUniform,
+    ds.PriorFunction.ert_erf: PriorErtErf,
+    ds.PriorFunction.ert_derf: PriorErtDErf,
+}
+
+
+def _prior_to_dict(prior: ds.Prior) -> dict:
+    return (
+        PRIOR_FUNCTION_TO_PYDANTIC[prior.function]
+        .parse_obj(
+            {key: val for key, val in zip(prior.argument_names, prior.argument_values)}
+        )
+        .dict()
+    )
+
+
+def experiment_priors_to_dict(experiment: ds.Experiment) -> Mapping[str, dict]:
```

Comment:
Sorry, forgot to post this one, i guess this should be private as well? Everything except the functions annotated with `@router` i guess is meant being private?

Reply:
Also this's used in other places too.